RadarSDK/Radar.m

```diff
@@ -28,10 +28,16 @@ + (id)sharedInstance {
 
 + (void)initializeWithPublishableKey:(NSString *)publishableKey {
     [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Initializing"];
```

Comment:
Shall we add a note to encourage the developers to call this method within their 'didLaunchWithOptions' method?

Reply:
Yup, it's in the docs: https://radar.io/documentation/sdk/ios#initialize-sdk